ci/owncloud/upload_to_owncloud.sh

```diff
@@ -28,8 +28,24 @@ usage() {
     echo ""
 }
 
+create_dirs() {
+    # Recursively create directory path in remote
+    # $1 remote path to create
+    local path=
+    for dir in $(echo "$1" | tr "/" "\\n"); do
+        if [ -z "$path" ]; then
+            create_dir "$dir"
+            path="$dir"
+        else
+            create_dir "$path/$dir"
+            path="$path/$dir"
+        fi
+    done
```

Comment:
You can also loop over $1 directly if you set IFS to '/':
```suggestion
    local path IFS
    path=
    IFS='/'
    for dir in $1; do
        if [ -z "$path" ]; then
            create_dir "$dir"
            path="$dir"
        else
            create_dir "$path/$dir"
            path="$path/$dir"
        fi
    done
    unset IFS
```
(`unset IFS` is only there "just in case" the shell do not implement `local` i.e. it's not needed for bash)

Reply:
Done